Merchant/App/Settings/SettingsViewController.swift

```diff
@@ -220,27 +220,21 @@ extension SettingsViewController: UITableViewDelegate, UITableViewDataSource {
 }
 
 extension SettingsViewController: UITextFieldDelegate {
-    
     func textFieldShouldReturn(_ textField: UITextField) -> Bool {
         return textField.resignFirstResponder()
     }
-
+    
```

Comment:
@Mattijah **[MUST]**
Unneeded space. Please remove it

Reply:
This offset was (is) added automatically by Xcode for every blank line. I can remove it but I feel like there are many more lines that have or don't have this. I think it would be nice to fix them all in 1 commit across the entire project and add the preferred settings to some kind of a swift-style-guide.